docs/configuring.md

```diff
@@ -14,6 +14,22 @@ Users may set Pixeebot preferences at either the organization or repository leve
 2. **Organization-wide preferences:**
    Alternatively, you can create the `pixeebot.yaml` file in the `.github` directory of your `.github` repository. This will serve as the global preferences that apply to all repositories in this installation.
 
+## Configuring Catalogs
+Pixeebot offers three levels of catalogs for code improvements: less, moderate, and more.
+
+- less: Pixeebot will only suggest improvements for findings identified by a third-party security tool. See the [third-party security tools](code-scanning-tools) page for more information.
+- moderate: In addition to the third-party security tool findings, Suggested fixes will also be provided for security findings identified by Pixeebot.
+- more: Includes everything from the moderate level, along with additional code improvements identified by Pixeebot such as style improvements and best practices.
```

Comment:
- **More:** Includes everything from the Less and Moderate catalogs, along with additional suggestions Pixeebot identifies, such as style improvement and best practices.